cmd/server-startup-msg.go

```diff
@@ -249,8 +250,12 @@ func printObjectAPIMsg() {
 // Get formatted disk/storage info message.
 func getStorageInfoMsgSafeMode(storageInfo StorageInfo) string {
 	var msg string
+	var mcMessage string
 	if storageInfo.Backend.Type == BackendErasure {
-		diskInfo := fmt.Sprintf(" %d Online, %d Offline. ", storageInfo.Backend.OnlineDisks.Sum(), storageInfo.Backend.OfflineDisks.Sum())
+		if storageInfo.Backend.OfflineDisks.Sum() > 0 {
+			mcMessage = "Run `mc admin info` to look for latest info`"
```

Comment:
Perhaps we have to say 
```suggestion
			mcMessage = "Run `mc admin info` to look for server/disk info`"
```